orga/app/routes/authenticated/campaigns/participant-assessment/results.js

```diff
@@ -1,8 +1,12 @@
 import Route from '@ember/routing/route';
 
 export default class ResultsRoute extends Route {
-  model() {
+  async model() {
     const { campaignAssessmentParticipation } = this.modelFor('authenticated.campaigns.participant-assessment');
-    return campaignAssessmentParticipation;
+    const campaignAssessmentParticipationResult =
+      await campaignAssessmentParticipation.campaignAssessmentParticipationResult;
```

Comment:
**thought** 🤔  cette partie est assez nébuleuse pour moi... je viens de faire le chemin en suivant les modèles mais c'est très compliqué, je me demande si on ne devrait pas plutot recuperer le bon model directement

Reply:
Ouai, c'est des sous model de sous model. j'ai voulu éviter de remonter une grappe de model pour n'en n'utiliser qu'un bout. 

Ce serait peut être une bonne chose de revoir les routes orga. elles ont peut être trop d'informations

Reply:
on peut rajouter un ticket de dette technique en "to be prioritized"